Mods to add related_people to node--news template

hpleas opened the pull request:
Add related people link to News pages

hpleas committed:
Mods to add related_people to node--news template

trubach reviewed (changes requested):
Hey @hpleas - it looks like the spacing got switched from 2 spaces to 4 spaces. Can you switch it back to 2 spaces and resubmit? Thanks!

hpleas committed:
Drupal spacing correction

hpleas commented:
Hi @trubach,

I don't think this needs to be resubmitted.  It's pushed and you should be able to see the adjustments in the existing pull request.  Let me know if its in the format you would like.

Henry

trubach committed:
Update node--news--full.html.twig

removing featured image in commit to prevent duplicate image showing up when another patch is applied

trubach commented:
Closing - no longer needed with the updated node--news.html.twig template, which honors the view display.
